src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -1015,6 +1018,17 @@ export namespace SignatureHelp {
 	}
 }
 
+export namespace InlineHint {
+	export function from(hint: vscode.InlineHint): modes.InlineHint {
+		return {
+			text: hint.text,
+			position: Position.from(hint.position),
+			whitespaceBefore: hint.whitespaceBefore,
+			whitespaceAfter: hint.whitespaceAfter
+		};
+	}
```

Comment:
This needs a `to` counterpart is will be used for an API command so that other extension can execute this logic (without UI) as well.